### ISSUE TITLE
Reviews to be addressed

### ISSUE THREAD

**Issue by abbyturi** [UBC-MDS/traders_copilot_mzza_25#46]
- [x] #49
- [x] #50
running pytest tests/ --cov=traders_copilot_mzza_25 is not working for me. It looks like you may be missing this in your dependencies in your pyproject.toml file under [tool.poetry.group.dev.dependencies]
- [x] #51
- [x] #47
- [x] #52
- [x] #53

- [x] #58
- [x] #54
- [x] #55
- [x] #56
- [x] #57
Suggested change:
avg_gain = gain.ewm(span=window, adjust=False).mean()
avg_loss = loss.ewm(span=window, adjust=False).mean()
